db/migrate/20161231030437_create_days.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDays < ActiveRecord::Migration[5.0]
+  def change
+    create_table :days do |t|
+      t.references :calendar
+      t.string :image_url, null: false
+      t.string :location, null: false
```

Comment:
Never mind! I see te benefits in having the location required. It shouldn't be blank either for the Twitter handle calendar. Looks good. I'll merge!